narratives/ncov_sit-rep_2020-03-20.md

```diff
@@ -0,0 +1,2490 @@
+In fact, [the WHO reports](https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200319-sitrep-59-covid-19.pdf?sfvrsn=c3dcdef9_2) confirmed cases in 159 out of 195 total countries in the world.
+<br><br>
+However, not all introductions result in outbreaks.
+
+<!-- There is no right side text -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 8 -->
+# [Epidemics become evident ~4-8 weeks after introduction](https://nextstrain.org/ncov/2020-03-19?legend=closed&c=num_date&d=tree&f_division=Washington&label=clade:B1&p=full)
+
+It may seem as though COVID-19 has suddenly exploded.
+For many people, this outbreak may have gone from an abstract concern in other parts of the world to something that severely impacts their daily lives, seemingly overnight.
+<br><br>
+But, keep in mind that this has been brewing for awhile. It's not necessarily that the virus is being introduced to new places more often.
+Most likely, many places are starting to become aware of localized outbreaks that were started by introductions several weeks ago.
+<br><br>
+Areas that aren't yet aware of local outbreaks should prepare now and start surveillance testing.  
+
+<!-- This is the right-side text -->
+
+```auspiceMainDisplayMarkdown
+# How do introductions become outbreaks?
+
+Sometimes these introductions don't result in secondary cases or localized outbreaks -- especially if the index case is detected and isolated quickly.
+Many times, however, the virus spreads undetected through the local population until the outbreak reaches a significant size.
+
+Below is a thought experiment showing how this played out in Wuhan. On the y axis we have space; on the x axis, we have time. Shaded area represents the number of cases.
+
+"COVID-19 in Wuhan went from an index case in ~Nov 2019 to several thousand cases by mid-Jan 2020, thus going from initial seeding event to widespread local transmission in the span of ~10 weeks. We believe that international seeding events started to occur in mid-Jan. Thus we have a critical ~10 weeks from then to late-March to contain these nascent outbreaks before they become sizable." [- Trevor Bedford](https://twitter.com/trvrb/status/1226241284207038464), Feb 2020
+
+These new outbreaks may, in turn, seed further introductions to other locations.
+
+<img src="https://github.com/nextstrain/ncov/raw/master/figures/local-spark-expansion.jpeg" width="70%">
+
+
+```
+
+<!-- ############ SLIDE BREAK ############# -->
+# [How outbreaks grow: an example](https://nextstrain.org/ncov/2020-03-19?legend=closed&d=tree,map&f_division=Washington&label=clade:B1&p=grid&r=location)
+
+We can see a clear example of this in the data from Washington state.
+Mousing over root of the main clade shows that the virus was likely introduced to this area around the end of January 2020 ([methods](https://nextstrain.org/narratives/trees-background?n=6)); this is consistent with the first reported case on January 19, 2020.
```

Comment:
Mousing over root of the main clade shows that the virus was likely introduced to this area between the end of January 2020 and mid February ([methods](https://nextstrain.org/narratives/trees-background?n=6)).

I would drop: "this is consistent with the first reported case on January 19, 2020.". There are a bunch of samples from China that are the same as WA1 and were sampled later than WA1. It could be two introductions into WA